tools/cloud-build/daily-tests/create_deployment.sh

```diff
@@ -69,3 +73,6 @@ tar -czf "${DEPLOYMENT_NAME}.tgz" "${DEPLOYMENT_NAME}" ||
 		echo "could not tarball blueprint"
 		exit 1
 	}
+
+echo "Copying ${DEPLOYMENT_NAME}.tgz to gs://daily-tests-tf-state/${TEST_NAME}/"
```

Comment:
nit: `daily-tests-tf-state` is used 4 times in this file. It might be helpful to make it a constant and reuse it.

Reply:
my own side nit: we might prefer to have this set via an environment variable similar to project ID both for reasons of exposure in source and portability of the implementation

Reply:
Good point, I updated it.